pkg/identities/validation.go

```diff
@@ -50,7 +54,8 @@ func (p *PayloadValidator) Validate(ctx context.Context, method, endpoint string
 	if p.isCreateIdentity(method, endpoint) {
 		createIdentity := new(CreateIdentityRequest)
 		if err := json.Unmarshal(body, createIdentity); err != nil {
-			return ctx, nil, err
+			p.logger.Info("Json parsing error: ", err)
```

Comment:
This should be `p.logger.Error(...)`
same below

Reply:
Most of the time it will happen because someone sent a malformed json, so imho it is not worth logging it as an error. If you believe this will add value as an error, I have no problem to change it back.